packages/core/src/codewhispererChat/clients/chat/v0/chat.ts

```diff
@@ -141,6 +142,10 @@ export class ChatSession {
         }
 
         this.sessionId = response.conversationId
+        if (this.sessionId?.length === 0) {
+            getLogger().debug(`Session ID: ${this.sessionId} is empty. Generating random UUID`)
+            this.sessionId = crypto.randomUUID()
```

Comment:
Can we use https://github.com/aws/aws-toolkit-vscode/blob/master/packages/core/src/shared/crypto.ts#L30-L36

Reply:
+1

Reply:
Fixed